python/paddle/hapi/dynamic_flops.py

```diff
@@ -22,10 +29,21 @@
 
 from .static_flops import Table, static_flops
 
+if TYPE_CHECKING:
+    from paddle import Tensor
+    from paddle.nn import Layer
+    from paddle.static import Program
+_CustomOpsAlias: TypeAlias = Callable[[Callable[..., Any]], Callable[..., Any]]
```

Comment:
```suggestion
_CustomOpsAlias: TypeAlias = dict[Layer, Callable[..., None]]
```
貌似是个 dict ？

另外，缩进到 `TYPE_CHECKING` 里面吧 ～

Reply:
收到